source/_integrations/wled.markdown

```diff
@@ -103,3 +103,15 @@ This service allows for controlling the WLED effect.
 
 A list of all available effects (and the behavior of the intensity for each
 effect) [is documented in the WLED Wiki](https://github.com/Aircoookie/WLED/wiki/List-of-effects-and-palettes#effects).
+
+### Service `wled.preset`
+
+This service allows for loading a preset saved on the WLED device.
+
+| Service Data Attribute | Required | Description                                                                                                     |
+| ---------------------- | -------- | --------------------------------------------------------------------------------------------------------------- |
+| `entity_id`            | no       | A WLED entity ID to load the preset from.                                                                       |
+| `effect`               | no       | ID of the preset slot to load from. Number between `1` and `16`.                                                |
```

Comment:
Consider the change in the parent PR, let's remove the last bit on the `Number between`...

Reply:
Done. I also realized I had `effect` as the attribute name when it should be `preset` so I fixed that as well.